prow/config/templates/config-ConfigMap.yaml

```diff
@@ -52,15 +52,7 @@ data:
             config:
             name: buildlog
           required_files:
-          - build-log.txt
-        - lens:
-            name: junit
-          required_files:
-          - .*/junit.*\.xml
-        - lens:
-            name: podinfo
-          required_files:
-          - podinfo.json
+          - build-log.txt|controller_logs
```

Comment:
is this right? You pipe directories to be created to the `required_files` value?

Reply:
The documentation is ... nonexistant. From my tests this did work. I'm actually going to move it into the `optional_file` array, though. Since we may not reach that part of the script if tests fail early.